acquia_lift_profiles/acquia_lift_profiles.admin.inc

```diff
@@ -6,6 +6,84 @@
  */
 
 /**
+ * Admin form for configuring lift_event_values behavior.
+ */
+function acquia_lift_profiles_lift_event_values_form($form, &$form_state) {
+  $form = array(
+    '#attached' => array(
+      'css' => array(
+        drupal_get_path('module', 'acquia_lift') . '/css/acquia_lift.admin.css',
+      ),
+    ),
+  );
+
+  $form['lift_event_values'] = array(
+    '#tree' => TRUE,
+    '#theme' => 'acquia_lift_profiles_lift_event_value_form_table',
+  );
+  $lift_event_values = variable_get('acquia_lift_profiles_lift_event_values', array());
+  foreach(visitor_actions_get_actions() as $machine_name => $action) {
+    $action_values = isset($lift_event_values[$machine_name]) ? $lift_event_values[$machine_name] : array();
+    $engagement_score = isset($action_values['engagement_score']) ? $action_values['engagement_score'] : ACQUIA_LIFT_PROFILES_ENGAGEMENT_SCORE_DEFAULT;
+    $global_value = isset($action_values['global_value']) ? $action_values['global_value'] : ACQUIA_LIFT_PROFILES_GLOBAL_VALUE_DEFAULT;
+    $row_label = $action['label'];
+    // If the action has a machine_name, then it is a custom action and can have
+    // a "Edit" link.
+    if (isset($action['machine_name'])) {
+      $row_label .= ' ' . l(
+        t('Edit'),
+        'admin/structure/acquia_lift/visitor_action/' . $machine_name . '/edit',
+        array(
+          'attributes' => array(
+            'class' => array(
+              'ctools-use-modal',
+              'ctools-modal-acquia-lift-style',
+              'acquia-lift-configure',
+            ),
+            'id' => 'edit-visitor-action-' . $machine_name,
+          ),
+        )
+      );
+    }
+    $row_engagement_score = array(
+      '#type' => 'textfield',
+      '#size' => 5,
+      '#title' => NULL,
+      '#default_value' => $engagement_score,
+      '#element_validate' => array('element_validate_integer_positive'),
+    );
+    $row_global_value = array(
+      '#type' => 'textfield',
+      '#size' => 5,
+      '#title' => NULL,
+      '#default_value' => $global_value,
+      '#element_validate' => array('element_validate_integer_positive'),
+    );
+    $form['lift_event_values'][$machine_name] = array(
+      '#label' => $row_label,
+      'engagement_score' => $row_engagement_score,
+      'global_value' => $row_global_value,
+    );
+  }
+
+  $form['submit'] = array(
+    '#type' => 'submit',
+    '#value' => t('Save'),
+  );
+
+  return $form;
+}
+
+/**
+ * Submit handler for the lift event_values configuration form.
+ */
+function acquia_lift_profiles_lift_event_values_form_submit($form, &$form_state) {
+  $lift_event_values = $form_state['values']['lift_event_values'];
+  // No additional sanitization only because all the inputs are through element_validate_integer_positive()
+  variable_set('acquia_lift_profiles_lift_event_values', $lift_event_values);
+}
```

Comment:
Since there's no custom logic required here and the form values correspond exactly to the variable that should be stored, it might be possible to use system_settings_form()?